tests/event-bus/e2e-tester/e2e-tester.go

```diff
@@ -245,85 +243,75 @@ func initSubscriberUrls(subDetails *subscriberDetails) {
 }
 
 func shutdown(code int, subscriber *subscriberDetails) {
-	log.Println("Send shutdown request to Subscriber")
+	log.Info("send shutdown request to Subscriber")
 	if _, err := http.Post(subscriber.subscriberShutdownEndpointURL, "application/json", strings.NewReader(`{"shutdown": "true"}`)); err != nil {
-		log.Printf("Warning: Shutdown Subscriber falied: %v", err)
+		log.WithField("error", err).Warning("shutdown Subscriber failed")
 	}
-	log.Println("Delete Subscriber deployment")
+	log.Info("delete Subscriber deployment")
 	deletePolicy := metav1.DeletePropagationForeground
 	gracePeriodSeconds := int64(0)
 
 	if err := clientK8S.AppsV1().Deployments(subscriber.subscriberNamespace).Delete(util.SubscriberName,
 		&metav1.DeleteOptions{GracePeriodSeconds: &gracePeriodSeconds, PropagationPolicy: &deletePolicy}); err != nil {
-		log.Printf("Warning: Delete Subscriber Deployment falied: %v", err)
+		log.WithField("error", err).Warn("delete Subscriber Deployment failed")
 	}
-	log.Println("Delete Subscriber service")
+	log.Info("delete Subscriber service")
 	if err := clientK8S.CoreV1().Services(subscriber.subscriberNamespace).Delete(util.SubscriberName,
 		&metav1.DeleteOptions{GracePeriodSeconds: &gracePeriodSeconds}); err != nil {
-		log.Printf("Warning: Delete Subscriber Service falied: %v", err)
+		log.WithField("error", err).Warn("delete Subscriber Service failed")
```

Comment:
Default log level is `Info`. Hence warnings will get logged.

Reply:
No flag needed atm.

Reply:
As discussed in person: We are going to add a flag such that the user can choose to change the default level to e.g `debug`.

Reply:
As discussed in person: We are going to add it for the `e2e-tester.go` only atm, since introducing it in `upgrade` and `backup-restore` test means we should introduce it to all tests listed there. Otherwise it is not consistent (using the log level only for event-bus) tests. Going to add an issue for this